src/Components/Patient/ManagePatients.tsx

```diff
@@ -477,6 +479,8 @@ export const PatientManager = () => {
     [fetchFacilityBadgeName, fetchLocationBadgeName]
   );
 
+  const { data: permittedData } = useQuery(routes.getPermittedFacilities, {});
```

Comment:
variable name `permittedData` doesn't make sense right?

```suggestion
  const { data: permittedFacilities } = useQuery(routes.getPermittedFacilities, {});
```